services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationForm.tsx

```diff
@@ -0,0 +1,378 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { useDispatch, useSelector } from "react-redux";
+import { Field, change, getFormValues } from "redux-form";
+import "@ant-design/compatible/assets/index.css";
+import { Col, Row, Typography, Button } from "antd";
+import { required } from "@common/utils/Validate";
```

Comment:
Would do a little more clean up in these imports:
- combine imports from the same source, namely from strings.
- kill the @ant-design/compatible/assets/index.css outdated stylesheet
- remove unused imports (sonarcloud is pointing out `required` is unused)